scvi/module/_multivae.py

```diff
@@ -55,13 +57,27 @@ class MULTIVAE(BaseModuleClass):
         Number of input regions.
     n_input_genes
         Number of input genes.
+    modality_weights
+        * ``"equal"``: Equal weight in each modality
+        * ``"universal"``: Learn weights across modalities w_m.
+        * ``"cell"``: Learn weights across modalities and cells. w_{m,c}
+    modality_penalty
+        * ``"Jeffreys"``: Jeffreys penalty to align modalities
+        * ``"MMD"``: MMD penalty to align modalities
+        * ``"None"``: No penalty
```

Comment:
Can you add the proper docstrings (these new parameters) as well in MULTIVI class?

Reply:
Also line 83 of _multivi.py should say MULTIVAE not PEAKVAE

Reply:
MULTIVAE, SOLVED.

ADDED DOCSTRING